includes/subscriptions/class-wc-payments-subscriptions-event-handler.php

```diff
@@ -55,10 +55,22 @@ public function __construct( WC_Payments_Invoice_Service $invoice_service, WC_Pa
 	public function handle_invoice_upcoming( array $body ) {
 		$event_object          = $this->get_event_property( $body, [ 'data', 'object' ] );
 		$wcpay_subscription_id = $this->get_event_property( $event_object, 'subscription' );
-		$wcpay_customer_id     = $this->get_event_property( $event_object, 'customer' );
```

Comment:
you'll notice that this `$wcpay_customer_id` has been removed. It wasn't being used.